Src/Web/Web.Nuget/Package.nuspec

```diff
@@ -24,14 +24,15 @@
       <group targetFramework="net45">
         <dependency id="Microsoft.ApplicationInsights" version="[$coresdkversion$]" />
         <dependency id="Microsoft.ApplicationInsights.WindowsServer" version="$version$" />
+        <dependency id="Microsoft.AspNet.TelemetryCorrelation" version="1.0.0-preview1-10407-02" />
```

Comment:
Do we need a Diagnostics Source here as well? Since we are using it explicitly - it will be nice to have it as a dependency to point to the specific version of it and allow nuget to do necessary versions resolutions and bindingredirects